src/components/Entry.vue

```diff
@@ -88,50 +88,51 @@
 import { validateID } from "@/services/validation.js";
 import { redirectToDestination } from "@/services/redirectToDestination.js";
 import { sendSQSMessage } from "@/services/API/sqs";
-import { typeToFunctionMap } from "@/services/basicValidationMapping.js";
-const authType = "SRN";
+import { validationTypeToFunctionMap } from "@/services/basicValidationMapping.js";
 
 export default {
-  name: "IdEntry",
+  name: "Entry",
   props: {
     redirectTo: String,
     redirectID: String,
     purpose: String,
     purposeParams: String,
-    textObject: Object,
-    inputObject: Object,
-    dataSourceObject: Object,
-    maxNumberOfInput: Number,
+    programData: Object,
     program: String,
     authType: String,
   },
   data() {
     return {
       userIDList: [{ userID: "", valid: false }], //array containing user-ids and a valid flag for each
-      invalidInputMessage: null, // whether the input is in correct format
       isCurrentUserValid: false, // whether the current user is valid
       validateCount: 0, // count the number of times the user has been validated
       invalidLoginMessage: "",
       isLoading: false,
+      invalidInputMessage: null, // whether the input being entered by the user matches the basic validation criteria
```

Comment:
The comment seems a little off
As it's a message, the comment should be something like "the message to show when the input being entered by the user does not match ..."